packages/app-store/routing-forms/pages/router/[...appPages].tsx

```diff
@@ -60,12 +61,11 @@ export const getServerSideProps = async function getServerSideProps(
 
   const response: Record<string, Pick<Response[string], "value">> = {};
   serializableForm.fields?.forEach((field) => {
-    const rawFieldResponse = fieldsResponses[getFieldIdentifier(field)] || "";
-    console.log(field);
-    const fieldResponse =
-      field.type === "multiselect" ? rawFieldResponse.split(",").map((r) => r.trim()) : rawFieldResponse;
+    const fieldResponse = fieldsResponses[getFieldIdentifier(field)] || "";
+    const value =
```

Comment:
Renamed fieldResponse as value(that better describes what it holds). Use, fieldResponse in place of rawFieldResponse.